src/main/target/SPRACINGF3/target.h

```diff
@@ -94,7 +94,7 @@
 #define ACC_MPU6050_ALIGN       CW270_DEG
 #endif
 
-#if defined(FLIP32F3OSD)
+#if defined(SPRACINGF3) || defined(FLIP32F3OSD)
```

Comment:
should we define `VARIANT` when compiling a variant so that we can then do this?
```
#if !defined(VARIANT)
...
#endif```

I don't like it that code for a variant appears above the original target!